### ISSUE TITLE
No experiment and data files for dataset fce97270-fce0-4744-8a4e-a93d95521852 in ENA 

### ISSUE THREAD

**Issue by ke4** [ebi-ait/hca-ebi-wrangler-central#544]
@ke4 has submitted the new replacement FASTQ files for this dataset (fce97270-fce0-4744-8a4e-a93d95521852) to ENA and asked to suppress the old BAM files.
ENA help desk accidentally suppressed the data in experiment level. @ke4 contacted with them and they are going to fix this issue and contact with @ke4 when they are ready.

This is the project link in ENA: https://www.ebi.ac.uk/ena/browser/view/PRJEB31821

**Comment by ke4** [comment]
Reply from ENA help desk:

Dear Karoly,

Thanks for contacting me via Slack about this suppression. Apologies for our
oversight in suppressing the Experiments here.

As I explained over slack - suppressing raw data via the experiment is
standard procedure as most runs are tightly coupled with experiments in a 1-1
ratio. However, we recognise that there are exceptions to this where multiple
runs are associated with the same experiment (like has been done here) and
this should have been identified prior to suppression - apologies for not
addressing this in this case.

I have marked the following Experiments and Runs for re-release (the set does
not include the runs that were requested to be suppressed in your email
below). Please review these accessions and let me know if you have any further
concerns:

ERR7124931 - ERX3266488
ERR7124932  - ERX3266487
ERR7124933 - ERX3266486
ERR7124934 - ERX3266489
ERR7124935 - ERX3266490
ERR7124936 - ERX3266491

These should become public again in the next 48 hours.

Best wishes,
Josie

**Comment by ke4** [comment]
My reply to ENA help desk:

Dear Josie,

I double checked and I can confirm that the above experiments and runs you listed in your email are the correct one to re-release.

On 1 Nov 2021, at 11:05, Josephine Burgin via RT <ena-helpdesk@ebi.ac.uk> wrote:

ERR7124931 - ERX3266488
ERR7124932  - ERX3266487
ERR7124933 - ERX3266486
ERR7124934 - ERX3266489
ERR7124935 - ERX3266490
ERR7124936 - ERX3266491


Many thanks for your extremely quick help.

Have a nice day,

Karoly

**Comment by ESapenaVentura** [comment]
tl;dr: @ke4 was in contact, there was an error, runs will re-appear in ~48 hours. Waiting on ENA.

**Comment by ESapenaVentura** [comment]
Runs have been re-released, moving to done :)
